include/lsst/afw/table/Schema.h

```diff
@@ -417,7 +417,7 @@ class SubSchema {
     /**
      *  @brief Implicit conversion to the appropriate Key type.
      *
-     *  Implicit conversion operators cannot be translated to Python.  Instead, the SWIG
+     *  Implicit conversion operators cannot be translated to Python.  Instead, the Python
      *  wrappers provide an equivalent asKey() method.
      */
     template <typename T>
```

Comment:
Then you might want to clarify in the comment that the issue is supporting assignments rather than e.g. passing to a method.